src/Comments.tsx

```diff
@@ -18,6 +19,8 @@ const Comments = () => {
   const [isDataLoading, setIsDataLoading] = useState(false);
   const [hasError, setHasError] = useState(false);
 
+  const { t } = useTranslation();
```

Comment:
it is better to write it at the top of the component

Reply:
Done.Thanks.